reinforcement_learning/bindings/cs/rl.net.cli/EntryPoints.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.IO;
+using Rl.Net;
+
+namespace Rl.Net.Cli {
+    static class EntryPoints
+    {
+        public static void Main(string [] args)
+        {
+            BasicUsageExample(args);
+        }
+
+        private static void WriteErrorAndExit(string errorMessage, int exitCode = -1)
+        {
+            Console.Error.WriteLine(errorMessage);
+            Environment.Exit(exitCode);
+        }
+
+        private static void WriteStatusAndExit(ApiStatus apiStatus)
+        {
+            WriteErrorAndExit(apiStatus.ErrorMessage);
+        }
+
+        // TODO: Pull this out to a separate sample.
+        public static void BasicUsageExample(string [] args)
+        {
+            const float outcome = 1.0f;
+            const string eventId = "event_id";
+            const string contextJson = @"({
+                                  ""GUser"":{""id"":""a"",""major"":""eng"",""hobby"":""hiking""},
+                                  ""_multi"":[ { ""TAction"":{""a1"":""f1""} },{""TAction"":{""a2"":""f2""}}]})";
+
+            if (args.Length != 1) {
+                WriteErrorAndExit("Missing path to client configuration json");
```

Comment:
Maybe the error message should specify that the argument is missing? The fact that the path is missing may not be that helpful.